app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -19,19 +19,23 @@ def confirm_request
     render locals: { hide_auth_buttons: true }
   end
 
+  def persist_auth_token(auth)
+    secret = SecureRandom.hex
+    secure_session.set "#{Users::AssociateAccountsController.key(secret)}", auth.to_json, expires: 10.minutes
+    "#{Discourse.base_path}/associate/#{secret}"
+  end
+
   def complete
-    auth = request.env["omniauth.auth"]
+    @auth = auth = request.env["omniauth.auth"]
```

Comment:
Does this need to be an instance variable instead of just a regular variable?

Reply:
🤦 you're right, it doesn't need to be an instance variable. Will fix.